APP/src/api.js

```diff
@@ -366,4 +378,64 @@ export const getPitScoutingData = async (year, event, team, callback) => {
   } catch (error) {
     console.error(error);
   }
+}
+
+export const getMatchScoutingData = async (year, event, team, callback) => {
+  try {
+    const storage_name = `${year}${event}_${team}_pitData`;
+    const data = null
+    if (data === null) {
+      const endpoint = `${API_ENDPOINT}/${year}/${event}/${team}/ScoutEntries`;
+      console.log("Requesting Data from: " + endpoint);
+      const response = await fetch(endpoint);
+      if (response.status == 200) {
+        const data = await response.json();
+        setWithExpiry(storage_name, data, default_ttl);
+        callback(data);
+      } else {
+        callback([]);
```

Comment:
If you are not getting 200 responses. You should probably throw an error or alert the user. This will help you debug later on.